ctru-sys/src/bin/docstring-to-rustdoc.rs

```diff
@@ -1,3 +1,24 @@
+//! This script transforms _some_ Boxygen comments to Rustdoc format
+//!
+//! # Usage
+//!
+//! ``cargo run --bin docstring-to-rustdoc -- [location of the bindings.rs]``
```

Comment:
I don't think \`\` does what you expect? To be marked as monospace/code, it only needs one \`

Reply:
I've always used \`\`, and ``cargo doc`` shows it fine. But I can change it

Reply:
Yeah I just read up on it and it works, but I've only ever seen the single version \` used.